pkg/config/parser.go

```diff
@@ -34,10 +34,29 @@ import (
 type LoaderConfiguration struct {
 	Seed int64 `json:"Seed"`
 
-	Platform string `json:"Platform"`
+	Platform       string `json:"Platform"`
+	InvokeProtocol string `json:"InvokeProtocol"`
+	YAMLSelector   string `json:"YAMLSelector"`
+	EndpointPort   int    `json:"EndpointPort"`
 
-	YAMLSelector string `json:"YAMLSelector"`
-	EndpointPort int    `json:"EndpointPort"`
+	DirigentControlPlaneIP   string `json:"DirigentControlPlaneIP"`
```

Comment:
You have some configuration for Dirigent setup in the additional file in the trace, some here. Can we have them in a single file?

Reply:
I think this can come as an independent PR.